openelis/WebContent/scripts/slickgrid/slick.formatters.js

```diff
@@ -49,8 +49,11 @@
     return "<span class='percent-complete-bar' style='background:" + color + ";width:" + value + "%'></span>";
   }
 
+
   function YesNoFormatter(row, cell, value, columnDef, dataContext) {
-    return value ? "Yes" : "No";
+  //var trans=MessageResources_pt.properties
```

Comment:
can we remove these comments if they are not being used?

Reply:
Yes Sure